client/webserver/site/src/js/markets.ts

```diff
@@ -802,6 +816,9 @@ export default class MarketsPage extends BasePage {
       Doc.bind(bttn, 'click', () => this.candleDurationSelected(dur))
       page.durBttnBox.appendChild(bttn)
     }
+
+    // load candlesticks here since we are resetting page.durBttnBox above.
+    this.loadCandles()
```

Comment:
I feel like `loadMarket` should still be called in `setMarket`, just moved to right after `this.setCandleDurBttns()`. Not a big deal though.